explainer.md

```diff
@@ -1,3 +1,122 @@
 ## Explainer
 
-### Examples
+### Introduction
+
+This document explains [extensions to the WEBRTC specification](https://w3c.github.io/webrtc-extensions). This document contains a number of sections, each of which contains design details to one specific extension.
+
+### RTCRtpContributingSource extensions on captureTimestamp and senderCaptureTimeOffset
+
+Authors:
+minyue@chromium.org
+hbos@chromium.org
+chxg@google.com
+
+Participate
+https://www.chromestatus.com/feature/5728533701722112
+
+#### Abstract
+
+This section explains the two new data fields in RTCRtpContributingSource, namely captureTimestamp and senderCaptureTimeOffset, see [here](https://w3c.github.io/webrtc-extensions/#rtcrtpcontributingsource-dictionary). They are introduced for audio-video synchronization and end-to-end delay measurements.
+
+The solution to audio-video synchronization and end-to-end delay measurements described here is particularly desired by systems, where an intermediate steam regenerator that terminates the streams originating from senders, an audio mixer as an example, is involved.
+
+#### Introduction
+
+In simple real-time communication systems, which involves only a sender and a receiver, the RTCP scheme [RFC3550] allows estimation of round-trip time and thus end-to-end delay.
+However, in a more sophisticated system, where an intermediate stream regenerator is involved, the estimation of end-to-end delay becomes more difficult. For example, if audio mixing is performed on the server side, the server terminates inbound media streams, processes the media data, and then generates new outbound media streams with fresh synchronization information.
+
+The solution proposed in this document is based on a new RTP header extension, namely absolute capture time, which contains two data fields:
+ * an absolute capture timestamp, which is stamped by the original capturer, and is supposed to be received untouched by the end receivers.
+ * an estimated clock offset with respect to the capturer's clock, which is supposed to be read and updated by every intermediate sender.
+
+With the absolute capture timestamps, end receivers can accurately measure the audio-video synchronization performance. With the `estimated clock offset`, which gets updated at each intermediate hop, end receivers can estimate their respective clock offset with respect to the capturer's clock, and then together with the absolute capture timestamp, measure the end-to-end delay. 
```

Comment:
Suggest replacing "the audio-video synchronization performance" with "how synchronized the audio and video tracks are". Target audience is not A/V experts.

Delete "respective".

Insert a comma after "and then" ("together with the absolute capture timestamp" is a subsidiary clause)